### ISSUE TITLE
About download Kitti dataset

### ISSUE THREAD

**Issue by yumianhuli2** [TRI-ML/sdflabel#10]
Hello!
Amazing work!
This page has many download datasets below：
[Download left color images of object data set (12 GB)](http://www.cvlibs.net/datasets/kitti/user_login.php)
    [Download right color images, if you want to use stereo information (12 GB)](http://www.cvlibs.net/datasets/kitti/user_login.php)
    [Download the 3 temporally preceding frames (left color) (36 GB)](http://www.cvlibs.net/datasets/kitti/user_login.php)
    [Download the 3 temporally preceding frames (right color) (36 GB)](http://www.cvlibs.net/datasets/kitti/user_login.php)
    [Download Velodyne point clouds, if you want to use laser information (29 GB)](http://www.cvlibs.net/datasets/kitti/user_login.php)
    [Download camera calibration matrices of object data set (16 MB)](http://www.cvlibs.net/datasets/kitti/user_login.php)
    [Download training labels of object data set (5 MB)](http://www.cvlibs.net/datasets/kitti/user_login.php)
    [Download object development kit (1 MB)](http://www.cvlibs.net/datasets/kitti/user_login.php) (including 3D object detection and [bird's eye view](http://www.cvlibs.net/datasets/kitti/eval_object.php?obj_benchmark=bev) evaluation code)
    [Download pre-trained LSVM baseline models (5 MB)](http://www.cvlibs.net/datasets/kitti/user_login.php) used in [Joint 3D Estimation of Objects and Scene Layout (NIPS 2011)](http://www.cvlibs.net/publications/Geiger2011NIPS.pdf). These models are referred to as LSVM-MDPM-sv (supervised version) and LSVM-MDPM-us (unsupervised version) in the tables below.
    [Download reference detections (L-SVM) for training and test set (800 MB)](http://www.cvlibs.net/datasets/kitti/user_login.php)
    Qianli Liao (NYU) has put together [code to convert from KITTI to PASCAL VOC file format](http://sourceforge.net/projects/etxml/) (documentation included, requires Emacs).
    Karl Rosaen (U.Mich) has released [code to convert between KITTI, KITTI tracking, Pascal VOC, Udacity, CrowdAI and AUTTI](https://github.com/umautobots/vod-converter) formats.
    We thank [David Stutz](http://davidstutz.de/) and [Bo Li](http://prclibo.github.io/) for developing the 3D object detection benchmark.
http://www.cvlibs.net/datasets/kitti/eval_object.php?obj_benchmark=3d

Which one do I need to download and place in the data/db/kitti directory？

Thank U！
